src/components/ScholarshipDetailCard.jsx

```diff
@@ -174,77 +163,55 @@ export default function ScholarshipDetailCard({ scholarship, preview }) {
       <Typography paragraph sx={{ whiteSpace: 'pre-line' }}>
         {description}
       </Typography>
-      <Box>
-        <DetailCardCell
-          label="Deadline"
-          text={deadline?.toLocaleDateString() || 'Unknown'}
-          bottom
-        />
-        <DetailCardCell
-          label="Award Amount"
-          text={ScholarshipAmount.toString(amount)}
-          bottom
-        />
+      <Box sx={{ mt: 6 }}>
+        <Typography variant="h5" component="h4" gutterBottom>
```

Comment:
I find this heading to be too close to the text below it.
![image](https://user-images.githubusercontent.com/8023233/145146257-1b31a986-87a2-4abe-b108-8884896f5cdb.png)

Maybe use `paragraph` instead of `gutterBottom`? If that's too much maybe reduce the `mt` value?

Reply:
`paragraph` gives it good space.